### PR TITLE
[HB-4461] Adding Missing Pieces of Documentation

### PR DESCRIPTION
**Pull request by SCastanedaMunoz**
# Description

Added missing documentation regarding Scoped Registries for UPM & Configuring Helium Unity SDK

### CODE REVIEW

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## COPPA
+COPPA Settings can be configured utilizing the following method:
+
+```csharp
+  HeliumSDK.SetSubjectToCoppa(true);
+  // or
+  HeliumSDK.SetSubjectToCoppa(false);
+```
+
+* By sending `setSubjectToCoppa (true)`, you indicate that you want your content treated as child-directed for purposes of COPPA. We will take steps to disable interest-based advertising for such ad requests.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:15]:
```suggestion
* By sending `SetSubjectToCoppa (true)`, you indicate that you want your content treated as child-directed for purposes of COPPA. We will take steps to disable interest-based advertising for such ad requests.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+* By sending `setSubjectToCoppa (false)`, you indicate that you don't want your content treated as child-directed for purposes of COPPA. You represent and warrant that your applications and services are not directed towards children and that you will not provide any information to Helium from a user under the age of 13.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:17]:
```suggestion
* By sending `SetSubjectToCoppa (false)`, you indicate that you don't want your content treated as child-directed for purposes of COPPA. You represent and warrant that your applications and services are not directed towards children and that you will not provide any information to Helium from a user under the age of 13.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## GDPR
+```csharp
+  HeliumSDK.SetSubjectToGDPR(true);
+  // or
+  HeliumSDK.SetSubjectToGDPR(false);
+```
+
+* By sending `setSubjectToGDPR (true)`, you indicate that GDPR is applied to this user from your application.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:26]:
```suggestion
* By sending `SetSubjectToGDPR (true)`, you indicate that GDPR is applied to this user from your application.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+* By sending `setSubjectToGDPR (false)`, you indicate that GDPR is not applied to this user from your application.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:28]:
```suggestion
* By sending `SetSubjectToGDPR (false)`, you indicate that GDPR is not applied to this user from your application.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+```csharp
+  HeliumSDK.SetUserHasGivenConsent(true);
+  // or
+  HeliumSDK.SetUserHasGivenConsent(false);
+```
+
+* By sending `setUserHasGivenConsent (true)`, you indicate that this user from your application has given consent to share personal data for behavioral targeted advertising.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:36]:
```suggestion
* By sending `SetUserHasGivenConsent (true)`, you indicate that this user from your application has given consent to share personal data for behavioral targeted advertising.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+* By sending `setUserHasGivenConsent (false)`, you indicate that this user from your application has not given consent to use its personal data for behavioral targeted advertising, so only contextual advertising is allowed.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:38]:
```suggestion
* By sending `SetUserHasGivenConsent (false)`, you indicate that this user from your application has not given consent to use its personal data for behavioral targeted advertising, so only contextual advertising is allowed.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+## CCPA
+```csharp
+  HeliumSDK.SetCCPAConsent(true);
+  // or
+  HeliumSDK.SetCCPAConsent(false);
+```
+
+* By sending `setCCPAConsent (true)`, you indicate that this user from your application has given consent to share personal data for behavioral targeted advertising under CCPA regulation.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:47]:
```suggestion
* By sending `SetCCPAConsent (true)`, you indicate that this user from your application has given consent to share personal data for behavioral targeted advertising under CCPA regulation.
```

File: com.chartboost.helium/Documentation/integration/configure.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+* By sending `setCCPAConsent (false)`, you indicate that this user from your application has not given consent to allow sharing personal data for behavioral targeted advertising under CCPA regulation.
```
Comment (bwised) [inline com.chartboost.helium/Documentation/integration/configure.md:49]:
```suggestion
* By sending `SetCCPAConsent (false)`, you indicate that this user from your application has not given consent to allow sharing personal data for behavioral targeted advertising under CCPA regulation.
```

**Comment by SCastanedaMunoz** [comment]
@bwised Nice catch 👍🏻

**Review by bwised — APPROVED** [review]

**Pull request merged**